fluent/src/resolver.js

```diff
@@ -123,6 +164,15 @@ function Tags(env, {name}) {
 /**
  * Resolve a variant expression to the variant object.
  *
+ * @param   {Object} env
+ *    Resolver environment object.
+ * @param   {Object} expr
+ *    An expression to be resolved.
+ * @param   {String} expr.id
+ *    An ID of a message for which the variant is resolved.
+ * @param   {String} expr.key
+ *    Variant key to be resolved.
```

Comment:
And `key` is either a number or a symbol: https://github.com/projectfluent/fluent.js/blob/41ea09d65cff11ef30afabe59afdcee480b0548a/fluent/src/parser.js#L690-L706 Any clue how to express this in terms of the docstring?